g/grpc-go/grpc-go_ubi_8.4.sh

```diff
@@ -0,0 +1,59 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : grpc-go 
+# Version       : v1.16.0, master
```

Comment:
"master" can keep changing. Can we find what is the latest version validated and mention the version number here

Reply:
Sure will update, the version tested is v1.41.0.